responsibleai/tabular/components/src/rai_component_utilities.py

```diff
@@ -130,22 +147,52 @@ def load_mltable(mltable_path: str) -> pd.DataFrame:
         tbl = mltable.load(mltable_path)
         df: pd.DataFrame = tbl.to_pandas_dataframe()
     except Exception as e:
-        _logger.info("Failed to load MLTable")
-        _logger.info(e)
+        _logger.info(f"Failed to load {mltable_path} as MLTable. ")
+        raise e
     return df
 
 
 def load_parquet(parquet_path: str) -> pd.DataFrame:
-    _logger.info("Loading parquet file: {0}".format(parquet_path))
-    df = pd.read_parquet(parquet_path)
+    _logger.info(f"Attempting to load {parquet_path} as parquet dataset")
+    try:
+        df = pd.read_parquet(parquet_path)
+    except Exception as e:
+        _logger.info(f"Failed to load {mltable_path} as MLTable. ")
```

Comment:
same thing here and below, this looks like a functional change

Reply:
yes, we want to copy/paste everything we changed from vNext repo to azureml-assets. So we can keep azureml-assets with latest RAI component updates